controller_manager:
  ros__parameters:
    update_rate: 500  # The rate (Hz) at which the controllers run

    position_controller:
      type: position_controllers/JointGroupPositionController

    effort_controller:
      type: effort_controllers/JointGroupEffortController

    joint_state_broadcaster:
     type: joint_state_broadcaster/JointStateBroadcaster

effort_controller:
  ros__parameters:
    joints: 
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7
      
    command_interfaces : 
      - effort
    state_interfaces : 
      - position 
      - velocity 

position_controller:
  ros__parameters:
    joints: 
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7

    command_interfaces : 
      - effort
    state_interfaces : 
      - position 
      - velocity 
    
